charts/substra-backend/templates/configmap-server.yaml

```diff
@@ -38,27 +38,61 @@ data:
     }
 
 ---
-
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ template "substra.fullname" . }}-server-uwsgi
 data:
   uwsgi.ini: |
     [uwsgi]
-    # chdir           = /usr/src/app
     module          = backend.wsgi
     env             = DJANGO_SETTINGS_MODULE=backend.settings.server.{{ .Values.backend.settings }}
     static-map      = /static=/usr/src/app/backend/statics
 
     master          = true
     processes       = {{ .Values.backend.uwsgiProcesses }}
     threads         = {{ .Values.backend.uwsgiThreads }}
-    http            = :{{ .Values.backend.service.port }}
-    # socket          = /tmp/substra-backend.sock
-    # chmod-socket    = 664
+    socket          = /usr/src/app/backend/socket/substra-backend.sock
+    chmod-socket = 666
+    # uid = www-data
+    # gid = www-data
+    vacuum          = true
 
     need-app        = true
     http-timeout    = 300
     socket-timeout  = 300
     harakiri        = 300
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "substra.fullname" . }}-server-nginx
+data:
+  nginx.conf: |
+    # user www-data www-data;
+    worker_processes  auto;
+
+    events {
+      # worker_connections  4096;  ## Default: 1024
+    }
+    http {
+
+      # the upstream component nginx needs to connect to
+      upstream django {
+          server unix:///usr/src/app/backend/socket/substra-backend.sock; # for a file socket
```

Comment:
This should be passed as a variable and also should be the path to statics below

Reply:
Those paths are hardcoded in deployment-server, I don't think we should let them configurable or we need to make everything configurable (path to statics, socket and medias) which is not the aim of this PR.
I keep it like this for consistency :)